### PR TITLE
remove video from programme page

### LINKED ISSUE #280: Update the video link
https://vimeo.com/188014263

### PR DESCRIPTION
**Pull request by sofer**
We don't want a video on the programme page. fixes #280

### CODE REVIEW

**Comment by sofer** [comment]
@des-des this is the one item that we need to fix before publishing the site, I think.

**Comment by sofer** [comment]
@nogainbar @bradreeder @franzmoro @ivanmauricio if anyone wants to grab this one, please do. I am hoping to publish the site as soon as it's merged.

File: _harp/programme/_overview.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-<iframe src="https://drive.google.com/file/d/0B-mliovLR9b_eHNGbDVIMVF3NkE/preview" width="720px" height="408px" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-
-## Programme
```
Comment (iteles) [inline _harp/programme/_overview.md]: Did you mean to delete the title here as well as the `iframe`?

Comment (sofer) [inline _harp/programme/_overview.md]: @iteles Yes I did. It is superfluous given the big header block above it.

**Pull request merged**